client/templates/app/initiativesUpdate/_initiativesUpdate.sass

```diff
@@ -0,0 +1,30 @@
+.initiatives-update
+  position: relative
+  // top: 0
```

Comment:
We can remove all the commented out lines before committing. git will preserve the history for us if we ever wanted to go back and re-add this.